packages/teams-components/stories/SplitButton/Default.stories.tsx

```diff
@@ -0,0 +1,240 @@
+import * as React from 'react';
+import { SplitButton } from '@fluentui-contrib/teams-components';
+import {
+  makeStyles,
+  tokens,
+  Menu,
+  MenuPopover,
+  MenuList,
+  MenuItem,
+} from '@fluentui/react-components';
+import {
+  CalendarRegular,
+  CalendarFilled,
+  bundleIcon,
+} from '@fluentui/react-icons';
+
+const CalendarIcon = bundleIcon(CalendarFilled, CalendarRegular);
+
+const useStyles = makeStyles({
+  sampleContainer: {
+    display: 'grid',
+    gridTemplateColumns: 'repeat(5, 120px)',
+    gap: tokens.spacingHorizontalL,
+  },
+
+  evil: {
+    background: 'red',
+  },
+});
+
+export const Default = () => {
+  const styles = useStyles();
+  return (
+    <div className={styles.sampleContainer}>
+      <Menu positioning="below-end">
+        <SplitButton>Split</SplitButton>
```

Comment:
Is it accessible to create a split button without a title/aria-label for the menu button part?